### PR TITLE
#187 move user rest controller logic

### PR DESCRIPTION
**Pull request by romach**
Move logic from `UserRestController` into services and fix old and add new test for this moving
Connect to #187

### CODE REVIEW

File: restservice/src/main/java/org/itevents/controller/UserRestController.java [hunk 1]
```diff
@@ -43,8 +38,7 @@
     @ResponseStatus(value = HttpStatus.OK)
     public TokenWrapper login(@ModelAttribute("username") String username,
                               @ModelAttribute("password") String password) {
-        String token = tokenService.createToken(username, password);
-        return new TokenWrapper(token);
+        return tokenService.createTokenWrapper(username, password);
```
Comment (IgorMaksymov) [inline restservice/src/main/java/org/itevents/controller/UserRestController.java]: i think this should be refactored. I think its better to make this a private method of tokenService.login(String username, String password)

**Comment by vaa25** [comment]
@romach 
1. `FilterWrapper.equals()` - redundant comment
2. `TokenWrapper` - redundant constructor
3. `UserRestController`, `UserService`,  `MyBatisEventService`, `EventService`- unused import
4. `JwtTokenService` - TokenWrapper is in controller package. Service can't use it. I think this part shouldn't change. Maybe I wrong.
5. Everywhere. for example `getEvent(int eventId)` -> `getEventById(int eventId)`. `getEvent` seems to be with `Event` as argument, not `int`
6. `MyBatisUserService`. `FilterConverter` the same as `TokenWrapper` (4)
@AndriyBaibak 30min

**Comment by romach** [comment]
@vaa25 thank you for review.
Errors are fixed.

**Comment by vaa25** [comment]
@romach thanks for corrections 
5. I mean every method in whole project. Yes, its not your fault and you may not to do it.
4,6. Wrappers and converters should be in controller package (see #121).

**Pull request closed without merging**